### PR TITLE
[reddit] Provide `title`, `content` and `bookmark-of` properties if present on a post

### LINKED ISSUE #212: [reddit] Is `content` the right mf2 field for a post?
For the title of the post, I'd expect `name` to be used, rather than `content`, as that makes it sound like it'd contain the full content for the post?

### PR DESCRIPTION
**Pull request by jamietanna**
Closes #212, #213.

### CODE REVIEW

File: granary/reddit.py [hunk 1]
```diff
@@ -137,7 +137,9 @@ def praw_to_object(self, thing, type):
     obj['url'] = self.BASE_URL + thing.permalink
 
     if type == 'submission':
-      obj['content'] = getattr(thing, 'title', None)
+      obj['title'] = getattr(thing, 'title', None)
```
[the remark this answers is not in the document]
Comment (jamietanna) [reply granary/reddit.py:140]: On which side, sorry?

```suggestion
      obj['title'] = getattr(thing, 'name', None)
```

Or

```suggestion
      obj['name'] = getattr(thing, 'title', None)
```

Or

```suggestion
      obj['name'] = getattr(thing, 'name', None)
```

File: granary/reddit.py [hunk 2]
```diff
@@ -137,7 +137,9 @@ def praw_to_object(self, thing, type):
...
+      obj['content'] = getattr(thing, 'selftext', None)
+      obj['bookmark-of'] = getattr(thing, 'url', None)
```
[the remark this answers is not in the document]
Comment (jamietanna) [reply granary/reddit.py:142]: That makes sense! I was trying to find where that was defined, will update it :+1:

**Comment by jamietanna** [comment]
@snarfed looks like the build is failing for changes made on the host infra / in the container, shall we get that fixed as a separate PR?

**Comment by snarfed** [comment]
> @snarfed looks like the build is failing for changes made on the host infra / in the container, shall we get that fixed as a separate PR?

i wish! it's actually spurious, started failing arbitrarily just today, unrelated to code changes. i think i've seen this failure once before, bad version of pip or setuptools. can't remember if i had to fix it somehow or if it was transient and cleared up on its own. i'll look.

**Comment by snarfed** [comment]
CI is fixed now, thanks for your patience, feel free to rebase on master and retry!

**Comment by jamietanna** [comment]
Also if we're happy with this, are you OK if I label with `hacktoberfest-accepted`?

**Comment by snarfed** [comment]
hey, sure! that label is totally fine with me.

File: granary/reddit.py [hunk 3]
```diff
@@ -137,14 +137,20 @@ def praw_to_object(self, thing, type):
...
       obj['tags'] = [
           {'objectType': 'article',
            'url': t,
            'displayName': t,
            } for t in util.extract_links(getattr(thing, 'selftext', None))
         ]
+
+      if getattr(thing, 'url', None):
+          obj['objectType'] = 'bookmark'
+          obj['targetUrl'] = getattr(thing, 'url')
```
Comment (snarfed) [inline granary/reddit.py:152]: minor nit: this can be just `thing.url`. not strictly necessary for this PR though.

**Pull request merged**